### PR TITLE
 Add how-to for RPM install on Leap base #421

### LINKED ISSUE #421: Add how-to for RPM install on Leap base method
This issue stems from a conversation with @phillxnet on the rockstor-website repo: https://github.com/rockstor/rockstor-website/pull/61#issuecomment-1460494019

We indeed currently have instructions for how to install Rockstor using its rpm on a Leap base install. These should *at least* also be present in our docs as a how-to. We could thus use this how-to in our docs as a single point of reference and point to these instructions wherever is needed (such as in our website downloads page, for instance).

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fixes #421 
@phillxnet, ready for review.

### This pull request's proposal
We currently have these instructions in the Downloads page of our website.
It would be beneficial to host these instructions as a how-to in our docs instead.

@phillxnet, I was wondering whether to link to this new how-to from our current sections related to Installation. However, given we do not recommend or support these instructions, I ended up refraining myself from doing so. This new document is of course listed in the menu/table of contents, though. Let me know if you would prefer seeing it linked to directly from somewhere in the docs.

### Checklist
- [x] With the proposed changes no Sphinx errors or warnings are generated.
- [x] I have added my name to the AUTHORS file, if required (descending alphabetical order).

### CODE REVIEW

**Comment by FroggyFlox** [comment]
Failure in `linkcheck`:
```
 (       data_loss: line   97) broken    https://btrfs.wiki.kernel.org/index.php/Glossary - 404 Client Error: Not Found for url: https://archive.kernel.org/oldwiki/btrfs.wiki.kernel.org/index.php/Glossary.html
```

@phillxnet, apologies if you've already told me that (I feel I heard it recently), but the Btrfs wiki has moved from kernel.org to only reside at https://btrfs.readthedocs.io.
We have the following:
```
$ grep "btrfs.wiki.kernel.org" *.rst
data_loss.rst:    Btrfs manages redundancy on a `Chunk <https://btrfs.wiki.kernel.org/index.php/Glossary>`_,
data_loss.rst:    not just a device level. See the btrfs `SysadminGuide <https://btrfs.wiki.kernel.org/index.php/SysadminGuide>`_
data_loss.rst:    Please see the `btrfs wiki status page <https://btrfs.wiki.kernel.org/index.php/Status>`_
data_loss.rst:`btrfs mailing list <https://btrfs.wiki.kernel.org/index.php/Btrfs_mailing_list>`_.
faq.rst:<https://btrfs.wiki.kernel.org/index.php/Main_Page>`_ for up to date
```

Only the first one fails; the others have permanent redirect to an archived version of the wiki. Given we need to first check that the contents are the same between the archived wiki at kernel.org and the new at readthedocs.io, I think it deserves it own issue to fix all of these links appropriately.

**Comment by phillxnet** [comment]
@FroggyFlox Re:
> ... but the Btrfs wiki has moved from kernel.org to only reside at https://btrfs.readthedocs.io.

Yes, I did change one of our links a little while back and they seemed to be mid-move at that time. As you say we can do that under it's own pr. Having a look at what you've done here now.

File: howtos/rpm_install.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+.. warning::
+
+    **For advanced users only**
+
+    This install method is predominantly for developers or advanced users and will likely require additional maintenance.
+    It may also serve as an option where a SLES base is required. Please note our undesirable disabling of apparmor.
+    We hope to resolve this partial failure in due time.
+
+The procedure, not recommended or supported, consists in adding the relevant Rockstor repositories to a dedicated vanilla openSUSE / SuSE SLES non transactional server install.
```
Comment (phillxnet) [inline howtos/rpm_install.rst:17]:
```suggestion
The procedure, not recommended or supported, consists of adding the relevant Rockstor repositories to a dedicated vanilla openSUSE / SuSE SLES non transactional server install.
```

Comment (phillxnet) [reply howtos/rpm_install.rst:17]: Trivial change likely not worth holding up the pr merge and publish I think. But noting for later reference.

**Review by phillxnet — APPROVED** [review]
@FroggyFlox It was a good to move/enhance these instructions form our downloads page. I'm all-on-board now. As-is reveiw-wise this all looks dandy (bar a trivial wording we can pop in another time) and so I'll move to merge and publish with what we have here.

Thanks for suggesting/seeing to this upgrade. Much appreciated. It will help to declutter our downloads page in time also: hence getting this in so we can link to it.

**Pull request merged**